tools/cldr-code/src/main/java/org/unicode/cldr/util/PathDescription.java

```diff
@@ -143,9 +143,9 @@ public class PathDescription {
                     + ".\n"
                     + "^//ldml/characters/exemplarCharacters$"
                     + RegexLookup.SEPARATOR
-                    + "Defines the set of characters used in your language. You may not edit or vote on this item at this time. Before filing any tickets to request changes, be sure to read "
-                    + CLDRURLS.EXEMPLAR_CHARACTERS
-                    + ".\n"
+                    + String.format(
```

Comment:
I don't think this was necessary (filed https://unicode-org.atlassian.net/browse/CLDR-16755) but also not a blocker.